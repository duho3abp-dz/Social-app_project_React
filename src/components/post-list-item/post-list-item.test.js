import React from 'react';
import {shallow} from 'enzyme';
import PostListItem from './post-list-item';

describe('Testing <PostListItem/>', () => {
    const postListItem = shallow(<PostListItem/>);

    describe('Testing snap', () => {
        it('PostListItem have rendered correctly', () => {
            expect(postListItem).toMatchSnapshot();
        });
    })
});