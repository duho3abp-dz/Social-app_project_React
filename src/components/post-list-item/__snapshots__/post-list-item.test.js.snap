// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <PostListItem/> Testing snap PostListItem have rendered correctly 1`] = `ShallowWrapper {}`;
